lighthouse/classes/events/source_plate_event.py

```diff
@@ -0,0 +1,6 @@
+from lighthouse.classes.plate_event import PlateEvent
+
+
+class SourcePlateEvent(PlateEvent):
+    def __init__(self, name: str) -> None:
+        super().__init__(name=name, plate_type=PlateEvent.PlateTypeEnum.SOURCE)
```

Comment:
I think this class is not needed at this moment and instead we can inherit directly from PlateEvent instead of inheriting from this; at now is an empty class with no other functionality and adds a new level of inheritance. I would remove it until is needed.

Reply:
As discussed, `plate_type` can be moved into specific events.